iquip/apps/dataviewer.py

```diff
@@ -316,13 +318,23 @@ def __init__(self, parent: Optional[QWidget] = None):
             spinbox.setReadOnly(True)
             spinbox.setFrame(False)
             self.valueBoxes[dataType] = spinbox
-            layout.addWidget(button, dataType, 1)
-            layout.addWidget(QLabel(":", self), dataType, 2)
-            layout.addWidget(spinbox, dataType, 3)
+            layout.addWidget(button, dataType, 2)
+            layout.addWidget(QLabel(":", self), dataType, 3)
+            layout.addWidget(spinbox, dataType, 4)
+        layout.setColumnStretch(1, 1)
         self.setDataType(DataPointWidget.DataType.P1)
+        # histogram viewer
+        self.histogram = HistogramViewer(title="Photon count histogram",
+                                         labels={"left": "#samples"})
```

Comment:
Is it correct that in order to make the bottom label in [this](https://github.com/snu-quiqcl/iquip/pull/151#issuecomment-1724835467), we need to set `labels={..., "bottom": "Photon count"}`?

Reply:
It will be set when we call `setData()`!

Reply:
Now, I understand it thanks for Jiyong's explanation!